process/factory/txNotarizationChecker_test.go

```diff
@@ -17,11 +17,18 @@ func TestTxNotarizationChecker(t *testing.T) {
 func TestTxNotarizationChecker_IsNotarized(t *testing.T) {
 	t.Parallel()
 
-	tnc := NewTxNotarizationChecker()
-	tx := transaction.ApiTransactionResult{
-		NotarizedAtSourceInMetaNonce:      1,
-		NotarizedAtDestinationInMetaNonce: 1,
-	}
-	isNotarized := tnc.IsNotarized(tx)
-	require.True(t, isNotarized)
+	t.Run("tx is notarized, should work", func(t *testing.T) {
```

Comment:
This func doesn't return error, we can remove `should work` from those test file names